### DISCUSSION TITLE
[Suggestion] - Monika's 21st Birthday

### DISCUSSION THREAD

**Discussion by ZapThundr** [Monika-After-Story/MonikaModDev#7902] [issue discussions]
Hello everyone. I think this is where you post suggestions correct? If not, then I am sorry in advance.

So I had an idea, since Monika is technically turning 21 this month I was thinking maybe for this special occasion, if it isn't already planned that is, she should be able to have a drink. I understand not everyone with Monika After Story is the/over the age of 21 but I'd think it'd be pretty cool if players who are of the age and maybe with high affection perhaps could be able to share a drink with her. In my mind I was thinking maybe a glass of wine would fit the scene well especially since 
 **SPOILER ALERT?**
she mentions Yuri bringing a bottle of wine to the club and how she was secretly wanting to try it. I'd understand if the idea is rejected especially since it's so close to her birthday and I don't have a clue how long it'll take to implement it.

Also I'd think it'd be pretty rad to have an "I feel" and have something like I had a bad dream/nightmare if it hasn't been suggested yet.
Anyway thank you all for reading this and I hope ya'll have a good day night.
Also Awesome jobs everyone on the mod!

**Comment by Freshairkaboom** [comment]
I can see a glass of wine being a nice base game gift we can only give Monika after her 21st birthday, after seeing the topic about Yuri bringing a bottle of wine. It's easy also to perform an 18+ check for her to accept it (like, if you try to give it to her while being underage, she asks where you got it instead). That way, everyone involved is an adult in most countries at least.

The reason I think it should be a gift instead of a story event on her birthday, is that well, not everyone likes alcohol consumption. I know it's very, very common and all, and yes you won't find many doctors who refuse you if you say you want a glass of wine, but some people have a history with alcohol that is not healthy, be it from parents or their society, or a personal problem with drinking alcohol.

That's why I think while it has merits to enter the base game, it should be optional, and that's why I think a gift fits the occasion. It is a consumable item, like coffee after all.

If you are 18+ and gift on special occasions like on her birthday, Christmas, Valentine's etc. then you would gain affection, while if you gift it on any other day she will politely decline and respond depending on the reason she declined, like she doesn't like to drink alcohol except on special occasions, which I think fits with her approach in the Yuri topic, like she's more curious to try it than drinking it regularly. She's also very health-focused, so her drinking wine on the regular doesn't seem like something she would do, if only to not tempt us to start.

Also, don't worry about time, there is still plenty of time before her birthday to get this done if we focus on it. I just need to confirm that this is something more people want first. I don't know how to design a wine glass sprite though. Maybe the gift itself should be a bottle and she pours it into a glass?

**Comment by Delcos** [comment]
Yes please.

**Comment by fromulus** [comment]
_Oh boy, here comes the fun police._

I could see wine (or any alcohol) maybe work as a one time only gift but nothing more.
The thing is, Monika is a girl who always puts a lot of importance on staying as healthy as possible, and she knows very well that drinking alcohol is on the top list of unhealthy habits to get.
Not to mention she (and by extension, we) wouldn't under any circomstance want to make the player feel like they're being incited to try drinking as well if they aren't already.

I personally kind of see it like that one time she tried cutting herself.
Monika is a curious girl, hence why when she learned about Yuri's habit, the curiosity eventually took over her and she gave it a try.
She knows, however, that this is a very self-destructive thing to do, and caring about her health as well as knowing she has an image of a responsible person to maintain, she promised herself (and later, to us) to never do it again.

I think alcohol should work the same.
Upon us gifting her a bottle (or just a glass actually, just to make sure she doesn't have too much at once) of wine, she'd be curious enough to try it, from which there's two possibilities which in my opinion would lead to the same end result :

Either she dislikes it, in which case she has no reason to want to drink it again;
Or she actually likes it, in which the possibility of her getting addicted starts to scare her.

Either way, she thanks the player for allowing her to have this experience, but asks them not to gift her any alcohol again, and refuses any further attempt at gifting more of it. 

There's also the fact that I think we should avoid as much as possible to have her drink because I honestly believe getting accidentally drunk could be a traumatizing experience for her.

Monika likes having control;
Over her self image, over her popularity with others, over the game.
It brings her a sense of comfort and lets her hide from her insecurities.

So what would happen if she gets wasted and starts losing control over everything, including her own mind and body ?
It'd be terrifying to go through for her.
I might argue that this could also trigger PTSD over another situation where she completely loses control;
The screaming void.

So, yeah.
I'm not saying alcohol in MAS is a big no-no, but for Monika's health, the influence she has over the player, and to avoid accidentally getting her to freak out completely, I'd much rather this would an extremely rare (as in, not even on every special occasion), if not unique occurence.

**Comment by Freshairkaboom** [comment]
Interesting, I agree with everything you said. It should probably be on very, very rare occurences or just a one off. Maybe she tries a sip and says she doesn't like it? Then she can explain that even if she would like the taste, she wouldn't want it as a habit. 

I was planning on having an alcohol related topic on around Christmas, talking about children sometimes feeling unsafe around drunk relatives and feeling isolated. This could be another occurence of this talk but more general?

**Comment by winika** [comment]
I would like to just drop this in aswell, drinking ages are usually different for some countries. So while 21 and older is the age in the United States. Up in Canada iirc our drinking age is 19 and older,  i just thought i'd add that in cause i know not everyone here is American. . Not bashing the idea, tho i also feel she wouldnt be that into alcohol unless you and her were spending a formal date together or something. Even then she would only have something light i feel.

**Comment by Freshairkaboom** [comment]
Yeah it would probably be best to wait until she is 21 just for that reason, so we can include every country and be sure none of them still consider her underage. In Norway you can drink at 18, but hard liquor is 21.

**Comment by multimokia** [comment]
Honestly, I genuinely don't see Monika as the type to drink. Maybe one time to try it, but I can't see her sticking with it. I also agree with a lot of what Fromulus said.

Imo this just feels more out of character than anything. Still not sure how much I even like the option to gift though.

**Comment by Retrolovania** [comment]
I'm not so sure if I want to see Monika getting involved with alcohol. I....it just doesn't feel right to me.

Plus, remember how she said that she doesn't have the same concept of time as we do? Even if she technically turns 21, her physical form can't age, so she's stuck at 18, which is underage even by Canadian standards. So, not only would it be bad for her health, but she'd also probably feel uneasy at us gifting her something that could potentially hinder her health and well-being....(plus, the FBI would probably hunt us down for giving alcohol to an underage schoolgirl.)

**Comment by ashyashton** [comment]
If we're going with "she doesn't age so she's 18" then it's also fair to say that her body isn't changing at all, including in response to alcohol, meaning her drinking alcohol wouldn't be unhealthy. It isn't fair to say she's physically real enough for alcohol to be bad for her but also not physically real enough to be aging to the legal drinking age.

Also, a bit of a note on the legality argument: Technically speaking, under the correct circumstances, you CAN drink "underage" in the US while still being legal! I'm gonna go off about that here because I think it's very interesting, just because of the workarounds that are legally in place. I'm going to go off a bit about the legality of it and the health concerns, but mostly the legality, just because I think the topic is really nifty. This is not me trying to force anyone to agree with me 

The requirements for someone under 21 to legally drink are as follows: A guardian/family member has to give permission and be present, it has to be in a private setting, and it cannot be enough for the person to get drunk. Let's break it down step-by-step.

Firstly, the requirement for family. Depending on the state you're in, it can differ a bit. In some places, the family member must be the one to give the alcohol. In others, the family member must be present while the drinking is occurring. And in others still, both are required! Now, one would think it has to be a parent, but it actually can be a parent, guardian, or spouse! The way I see it, there are two arguments that can be made here, those arguments being the guardian argument and the spouse argument.

Firstly, the guardian argument. Technically speaking, Monika does not have parents. She does not have a guardian at the beginning of the game. Legally, until she is an adult, she should be in foster care or being adopted. However, there's no one to adopt her. Technically, the player is the only other being in the world for her, which means guardianship could be said to have defaulted to the player if they are legally old enough for that. Which we're assuming they are because if they're not, they aren't old enough to buy the alcohol in the first place and it's all a moot point. So _technically speaking,_ an argument could be made that the player is Monika's guardian, even if the relationship currently in place functions far differently than that.

Next is the spouse argument. Now, we all know Monika's opinions on officially getting married before she comes to our reality. However, functionally speaking, the player could easily be considered her spouse even without an official marriage, especially given that she technically lives with the player since she lives in the computer _in the player's house,_ which means that it could easily be considered a common-law marriage at this point. It may feel like she's far away, but she does technically live in the player's house!

Which is a great segue into the next requirement, which is that it be in a private/home setting. One might argue that the classroom isn't private, but technically, the classroom is within the player's home, making it part of the player's home and private property. Also, even if that's pushing it too much, by now Monika definitely considers that place the closest thing she has to a home, and I think by now it just counts as being like an apartment. I don't know enough about squatter's rights to fully rely on them, but I feel like there's an argument to be made about that, as well.

And finally, it can't be enough to get her drunk. This actually segues into the health concerns! No, alcohol in excess is not healthy, but it's just like any other unhealthy thing. It's just like having lots of sugar, or fried foods, or heavily-processed snacks. They're not great for you, but in moderation, there's no true harm in any of it. And, if we are claiming she still counts as 18, then we now have a failsafe intact to encourage her to stay healthy about it! If she legally counts as 18, then she legally isn't allowed to drink enough to become inebriated, which means she'll be having maybe two glasses _at most._

I do understand people's concerns about health. But the fact of the matter is, it is nearly impossible to completely avoid unhealthy things. And Monika doesn't do that! She said so herself, in the "monika_lazy" topic:

> _"After a long day, I usually just want to sit around and do nothing. I get so burnt out, having to put on smiles and be full of energy the whole day. Sometimes I just want to get right into my pajamas and watch TV on the couch while eating junk food..."_

Monika strives to be healthy, yes. But she is not some perfect goddess who never does anything unhealthy. She even has issues with being regarded as such, as seen in the "monika_being_herself" topic. She states that she was looked at as, "some kind of unattainable goddess," and she was frustrated by this because she couldn't make friends without them thinking she was too perfect. By saying she can never, ever, _ever_ touch alcohol, not even a glass on her 21st birthday, all because it's unhealthy and she wants to be healthy? That's just doing exactly what her classmates did. You're shoving her into this mold of some pristine, perfect, always-healthy goddess, which she specifically states she has issues with!

I am not saying she should have to drink alcohol. But it _does_ make sense to have the option, at the very least. I agree with the people saying it should be a gift, thus making it completely opt-in. If you don't like the idea of Monika drinking alcohol, that's fine! You're welcome to not give her the gift! But for those of us who are aware that she's a person who isn't always healthy 10000% of the time, we can actually treat her like a human being* by being given the option. Alcohol isn't some devil's liquid or something that will kill you the second you drink it, and it shouldn't be treated like it is.

*This statement doesn't apply to people who don't want her drinking alcohol around them because of personal preferences/history with alcohol/etc., as this is totally valid reasoning for not utilizing the option. I'm specifically miffed with people treating her _the exact way she doesn't want to be treated._

**Comment by Delcos** [comment]
> _Oh boy, here comes the fun police._
> 
> I could see wine (or any alcohol) maybe work as a one time only gift but nothing more.
> The thing is, Monika is a girl who always puts a lot of importance on staying as healthy as possible, and she knows very well that drinking alcohol is on the top list of unhealthy habits to get.
> Not to mention she (and by extension, we) wouldn't under any circomstance want to make the player feel like they're being incited to try drinking as well if they aren't already.
> 
> I personally kind of see it like that one time she tried cutting herself.
> Monika is a curious girl, hence why when she learned about Yuri's habit, the curiosity eventually took over her and she gave it a try.
> She knows, however, that this is a very self-destructive thing to do, and caring about her health as well as knowing she has an image of a responsible person to maintain, she promised herself (and later, to us) to never do it again.
> 
> I think alcohol should work the same.
> Upon us gifting her a bottle (or just a glass actually, just to make sure she doesn't have too much at once) of wine, she'd be curious enough to try it, from which there's two possibilities which in my opinion would lead to the same end result :
> 
> Either she dislikes it, in which case she has no reason to want to drink it again;
> Or she actually likes it, in which the possibility of her getting addicted starts to scare her.
> 
> Either way, she thanks the player for allowing her to have this experience, but asks them not to gift her any alcohol again, and refuses any further attempt at gifting more of it.
> 
> There's also the fact that I think we should avoid as much as possible to have her drink because I honestly believe getting accidentally drunk could be a traumatizing experience for her.
> 
> Monika likes having control;
> Over her self image, over her popularity with others, over the game.
> It brings her a sense of comfort and lets her hide from her insecurities.
> 
> So what would happen if she gets wasted and starts losing control over everything, including her own mind and body ?
> It'd be terrifying to go through for her.
> I might argue that this could also trigger PTSD over another situation where she completely loses control;
> The screaming void.
> 
> So, yeah.
> I'm not saying alcohol in MAS is a big no-no, but for Monika's health, the influence she has over the player, and to avoid accidentally getting her to freak out completely, I'd much rather this would an extremely rare (as in, not even on every special occasion), if not unique occurence.

I'm from California, and I can tell you that wine (red especially) is a very common drink. You don't just drink it to drink, you have one or two glasses to enjoy it. It's nothing like hard liquor, and even has some health benefits.

**Comment by fromulus** [comment]
> I'm from California, and I can tell you that wine (red especially) is a very common drink. You don't just drink it to drink, you have one or two glasses to enjoy it. It's nothing like hard liquor, and even has some health benefits.

And I'm from France, where you'll see wine everywhere and where basically all the higher quality wine that's circulating in the world is made.
I still stand by everything I said.

All kinds of alcohol, including wine, _are_ dangerous no matter how you look at it, and it is not something Monika would take the risk of getting addicted to, neither is it something we want to normalize in the eyes of people playing the mod.


> Monika strives to be healthy, yes. But she is not some perfect goddess who never does anything unhealthy. She even has issues with being regarded as such, as seen in the "monika_being_herself" topic. She states that she was looked at as, "some kind of unattainable goddess," and she was frustrated by this because she couldn't make friends without them thinking she was too perfect. By saying she can never, ever, ever touch alcohol, not even a glass on her 21st birthday, all because it's unhealthy and she wants to be healthy? That's just doing exactly what her classmates did. You're shoving her into this mold of some pristine, perfect, always-healthy goddess, which she specifically states she has issues with!

I don't really understand your point here.
Her being seen as a "perfect goddess" isn't only due to her doing her best to stay healthy, it is due to her hiding herself behind a persona she's created for herself of a confident, responsible and "perfect" girl who's intelligent  and has no weaknesses whatsoever.

You can do your best to strive for being healthy (which is, like it or not, an actual part of Monika's character independant of her faked personality) and still be flawed in many other ways (which Monika is, such as her lack of comfort when it comes to dealing with people, her many insecurities she tries to hide even from us like believing she tends to ramble a lot, and so on).

So to consider that Monika, who as a reminder isn't even okay with drinking soda, smoking or eating junk food, would want to avoid alcohol as well isn't really more "shoving her into "this mold of some pristine, perfect, always-healthy goddess" than it is simply staying consistent with the actual, defined character that she is, in my opinion.
If anything, it'd be more faithful to her character to have her try to discourage the player from drinking alcohol than the other way around.

Like I said, there are flaws to Monika, and it is what I wanted to emphasize when writing `monika_being_herself`, but being unhealthy simply isn't one of them and there are more in-character and meaningful flaws to introduce to her character than having her drink alcohol.

Either way, getting her to drink, if it even makes it in, isn't going to be something that would happen regularly considering what I've seen of the opinions of the dev team and most contributors so far, and the whole point of an event where she tries out drinking would be to see her reactions.

This would imply making dedicated art such as a wine glass, which for a rare or unique occurence is more work than would be worth.

Quite frankly, the more I think about it, the more I stand with multimokia on the fact that this probably shouldn't become a thing at all.

**Comment by ashyashton** [comment]
>So to consider that Monika, who as a reminder isn't even okay with drinking soda, smoking or eating junk food, would want to avoid alcohol as well isn't really more "shoving her into "this mold of some pristine, perfect, always-healthy goddess" than it is simply staying consistent with the actual, defined character that she is, in my opinion.

Except... She literally has stated in the past that she does occasionally enjoy junk food, as I already stated. In the laziness topic, she herself states that she does occasionally indulge in junk food. As for the soda comment, she expresses concerns about drinking it in excess. And she's right, because just like with anything else, it should be in moderation. Soda has enough sugar in it that it should be treated like eating a candy bar or something: An occasional sweet treat to indulge in, not something to drink constantly. Just like alcohol, assuming there isn't already an addiction.

You are acting like drinking one glass of wine is the same as going cliff-diving or something. It isn't. And having one glass of wine is not going to get her addicted, so you can stop acting like it is.

Do you know what _else_ is an addictive substance? Caffeine. You know, like the kind she constantly drinks a ton of? If we're making her into some health nut who never does anything unhealthy, then we need to remove all coffee from the mod, because that addiction has clearly already formed and we should fix the problem now. It is no different. It is not healthy for you, but she drinks coffee anyway because she enjoys it, and drinking a glass of wine for special occasions is way less of a risk than multiple cups of coffee daily.

> This would imply making dedicated art such as a wine glass, which for a rare or unique occurence is more work than would be worth.

Okay, no more birthday cake, then? And birthday decorations? Because I can't think of a single other time that those assets are used in-game other than one specific occasion. How is a glass different? If anything, it should be easier because a wine glass is much more simple than a cake and a bunch of decorations. Also, the sprite would theoretically be able to be used more often, technically making it more useful than the other stuff.

I understand that you personally clearly have some issues with alcohol consumption, be it due to your own experiences or perhaps the experiences of a loved one, and that is valid. But it isn't fair to push beliefs onto other people, and that is what you are doing. You are essentially saying that because you don't want it in-game, no one should ever get the option, and that isn't fair. I think it should be added to the game, and anyone who feels the same way as you can feel free to just not use the feature. We aren't "normalizing" addiction, as you implied in your post. Things aren't just black and white, people can occasionally enjoy it without it being an addiction. The only thing we'd be "normalizing" is the idea that people can make choices about their own bodies for themselves.

**Comment by multimokia** [comment]
The fact that this is already so controversial from even a discussion standpoint doesn't bode well for adding this officially.

**Reply by ashyashton** [reply]
I don't love that this argument gets tossed around very frequently to shut down ideas but then ignored when it's convenient. Deleting .chr files was controversial but devs tried to shut down any and all criticism of it, but when it's something some players actually _want,_ then it being controversial is enough reason to not include it. Same argument has been used for LGBT issues, if I remember correctly. Why is it that when people want something added, controversy is enough to say no, but when people actively tell you they dislike something, that controversy is treated like people being whiny?

**Comment by ZapThundr** [comment]
Hmm I can kinda see why you'd say that MultiMokia, but I'd think just because it's "controversial" doesn't mean that it shouldn't be considered as bad or not implacable if that is what you are implying?

I do agree with Freshairkaboom comment  "The reason I think it should be a gift instead of a story event on her birthday, is that well, not everyone likes alcohol consumption. I know it's very, very common and all, and yes you won't find many doctors who refuse you if you say you want a glass of wine, but some people have a history with alcohol that is not healthy, be it from parents or their society, or a personal problem with drinking alcohol."
 Much better thought out then a special  event and having it be optional for the reasons stated above. 

I'd like to think Monika would enjoy it and I don't believe a glass of wine or two would hurt her and I don't think it's something considered unhealthy if given on some special occasions, ie Anniversaries, maybe birthdays, maybe New Year's, and Valentine's Day. It's more like a treat like chocolate and cupcakes and the lot, except a lot less often as she does like to be healthy.

I can also see why some of you might not want this to be implemented but I do think it should be optional and up to the player to decide to gift the bottle/glass in the first place and up to Monika if she would like to try it.  That way for the ones who don't want to give or if Monika doesn't like/want to try it then that's completely up to them but it wouldn't screw over the other players and Monika who would wanna try it, Especially since Monika has stated she secretly wanted to try wine, that should be another reason why the content should be added or at least considered.

I know we all have different views and believes, especially about how we view Monika and her personality but I don't think we
should push our believes about her on to others, whether you are for it or not.  So please keep it to pure discussion.

If you'd like for me to answer a question for some reason i'd be happy to do so.

Also I'd really like to hear everyone's opinions even if it doesn't get implemented.

**Comment by ZapThundr** [comment]
Also for the sprite if it does get added, I'd be more than willing to commission someone to create one if that's of any issue or concern.

**Comment by Legendkiller21** [comment]
Being optional is not a reason for this to be in the mod, either as an event or as a gift. There are things that should/will not be in the mod even as optionals.

**Comment by multimokia** [comment]
Again the point of beliefs *is* the issue here. Optional or not, if people know that xyz is her reaction to being given alcohol, it can --and will-- put people off.

This is a subject you'll never get everyone to agree on. Some don't see her drinking at all, some only see maybe one or two sips *ever* to try, and decide she doesn't want to, others may see a few sips on event days, and of course, there's people who have no problem at all with her drinking.

But when a reaction to something is known, it still very much upsets people who don't agree with it and find it out of character for her. This is the issue of it being controversial. I very much see this creating a divide in the community.

**Comment by lunulae** [comment]
I feel like this would be best as an opt-in submod if it is added. I have an interest in trying to still handle it in an in-character way so I might take it up in the future if there is a genuine interest, but I really don't want to try for a base-game addition.

My reasoning being that a lot of people have complicated relationships to alcohol. For some it's an important part of their lives in a moderate way, for others it's a source of trauma. I know we have sensitive mode, and a topic could moderate this, but like nsfw content I think this falls under something that could be more troublesome than the base sensitive-mode content.

On another note, we might consider Monika to be 21 if we consider her relation to DDLC's release date, but some people may disagree. For those in countries with higher  legal drinking ages I'd like to give them the courtesy of deciding this sort of canon for themselves if they started playing more recently.

**Comment by ThePotatoGuy** [comment]
leaving open as discussion

**Comment by Justformas** [comment]
Wow, I love the way @ashyashton thinks and argues. I think I agree with just about all, if not all of that. I was also aware about the under age 21 exceptions, and personally I don't like the whole 18 vs 21 distinction of adulthood for certain things within one country anyway. It's basically saying that 18 is an adult for many or most things, like going to war, voting, marriage and consent for sexual stuff, but not for other things like drinking alcohol. Even the exceptions may be kind of strange when you think about it, like it needs to be with a "guardian" or "spouse" that can act as a guardian if you're 18 to 20 years old, but that person might not be considered a guardian in that sense for anything else.

> Again the point of beliefs _is_ the issue here. Optional or not, if people know that xyz is her reaction to being given alcohol, it can --and will-- put people off.
> 
> This is a subject you'll never get everyone to agree on. Some don't see her drinking at all, some only see maybe one or two sips _ever_ to try, and decide she doesn't want to, others may see a few sips on event days, and of course, there's people who have no problem at all with her drinking.
> 
> But when a reaction to something is known, it still very much upsets people who don't agree with it and find it out of character for her. This is the issue of it being controversial. I very much see this creating a divide in the community.

@multimokia I like to believe that there can be compromise. I mean, it's not like everyone is always going to think that the way everything is implemented is perfect. That can be true of anything, but that doesn't mean that nothing should be implemented at all. So my belief is that it should be a gift, possibly only for special occasions (if 2 occasions are too close together in time, then maybe the player can only choose 1 to gift it on), and that she won't want or accept too much of it. Even more than one drink on the same occasion could be too much. She could explain why for reasons already stated and warn about addiction, so that she's not really encouraging you to drink or get addicted. I think this wouldn't really be inconsistent with Monika's character, since she is curious about things like that and the player would be encouraging her to try some.

In the wine topic, Monika also says that **"but we never bonded enough to get to that point..."** So there could also be an affection requirement, like 400 or 1000, so that Monika needs to love you a lot or feel more comfortable doing this with you. Like **"I wouldn't do this normally, but if it's with you..."** A hard limit like once or twice _ever_ though, while that might be more consistent with Monika's character, I think that might be too strict and would prefer the once in a while special occasion approach.

Also to quote @Retrolovania from another topic:

>People will find anything discomforting; it's just how humanity works.

But people who find Monika drinking at all to be discomforting don't have to give their Monika a drink, and she will never drink. Their Monika would not be acting "out of character" for them since the reaction would never happen, and she doesn't have to express any interest in trying alcohol outside of the wine topic as written and the reaction itself. They'd be free to treat the reaction as if it doesn't even exist for them, and not worry about other people wanting to do this with their own Monika.

People who want to be able to give Monika a drink shouldn't be restricted from doing that just because some people might not like the thought of Monika's reaction from being giving a drink, especially if those people don't want to give her a drink or see the reaction anyway.

On another note:

> a topic could moderate this, but like nsfw content I think this falls under something that could be more troublesome

Well, there are now some topics about sexual stuff in the base game, even if you don't count the ones that (might) only be triggered by reactions from going to certain websites. I'm thinking that the player probably has to indicate that they're at least 18 years old to get them. Those could maybe be considered nsfw, but maybe you only mean truly/blatantly nsfw like obscene language and explicit visuals.

Still, we have those topics in the game and the community has accepted how it has been done. So I'd like to think that there should also be a way to implement giving Monika a drink that the community can accept. Monika is already shown to not be infallible about health anyway, like saying that she occasionally consumes junk food (at least on some Fridays), regularly drinking coffee which can be addicting, and she likes mint ice cream. But she does generally seem to believe in moderation, and alcohol can be moderated too.
